src/test/java/com/cloudbees/jenkins/support/impl/NodeExecutorsTest.java

```diff
@@ -173,7 +173,7 @@ public void addContentsFiltered() throws Exception {
         WorkflowRun workflowRun = Optional.ofNullable(p.scheduleBuild2(0))
                 .orElseThrow(AssertionFailedError::new)
                 .waitForStart();
-        j.waitForMessage("Running on", workflowRun);
+        SemaphoreStep.waitForStart("wait/1", workflowRun);
```

Comment:
compare https://github.com/jenkinsci/support-core-plugin/blob/a457dfc405f57d37cc961db3bfbb3bba8b6cf3dd/src/test/java/com/cloudbees/jenkins/support/impl/NodeExecutorsTest.java#L172 and https://github.com/jenkinsci/support-core-plugin/blob/a457dfc405f57d37cc961db3bfbb3bba8b6cf3dd/src/test/java/com/cloudbees/jenkins/support/impl/NodeExecutorsTest.java#L197